src/binary/startupActionsHandler.ts

```diff
@@ -0,0 +1,15 @@
+import {BINARY_STARTUP_GRACE, MessageActions, StateType} from "../consts";
+import {openConfigWithSource} from "../commandsHandler";
+import {getStartupActions} from "./requests/startupActions";
+import {sleep} from "../utils";
+
+export default async function executeStartupActions(): Promise<void> {
+    await sleep(BINARY_STARTUP_GRACE);
+    const actionsResult = await getStartupActions();
+
+    if (actionsResult?.actions.includes(MessageActions.OPEN_HUB)){
+        return openConfigWithSource(StateType.STARTUP)();
+    }
+
+    return Promise.resolve();
```

Comment:
redundant, delete line (you are using async so by default the promise resolves

Reply:
IJ doesnt like it if i remove this line, plus i copied this code from `executeNotificationAction.ts`:

```typescript
export default async function executeNotificationAction(
  selectedActions: MessageActions[] | undefined
): Promise<void> {

  if (selectedActions?.includes(MessageActions.OPEN_HUB)){
    return openConfigWithSource(StateType.NOTIFICATION)();
  }
  return Promise.resolve();
}
```

Reply:
![image](https://user-images.githubusercontent.com/67855609/106158772-c72f1780-618c-11eb-9568-fc674e31f950.png)